### DISCUSSION TITLE
2021/11/28

### DISCUSSION THREAD

**Discussion by giscus** [doppelganger9/blog#258] [Post comments]
# 2021/11/28

A quick post explaining how we added GitHub Discussions to this blog's repository and will use it as an interactive comment section for each blog post.

http://localhost:3000/2021/11/28

**Comment by doppelganger9** [comment]
It works! Yay!

**Reply by doppelganger9** [reply]
![magic!](https://user-images.githubusercontent.com/316439/143790086-1e0bb089-6f87-4c9c-9eb3-a17762dee8dd.gif)

**Reply by abjutus** [reply]
![nice](https://c.tenor.com/KMxrZ-A6ev4AAAAC/nice-smack.gif)

**Reply by abjutus** [reply]
0.0
